### ISSUE TITLE
Qwen2-VL-2B cuda推理llm部分结果异常(已经关闭transformerFuse)

### ISSUE THREAD

**Issue by CYYAI** [alibaba/MNN#3362]
1. MNN版本使用3.0.5 
2. llmexport.py转mnn时去掉transformerFuse
@spinner_run(f'convert onnx model to ')
    def onnx2mnn(self, onnx_path, mnn_path, args = []):
        # convert_args = [
        #     '',
        #     '-f',
        #     'ONNX',
        #     '--modelFile',
        #     str(onnx_path),
        #     '--MNNModel',
        #     str(mnn_path),
        #     '--transformerFuse',
        #     '--allowCustomOp'
        # ]
        convert_args = [
            '',
            '-f',
            'ONNX',
            '--modelFile',
            str(onnx_path),
            '--MNNModel',
            str(mnn_path),
            '--allowCustomOp'
        ]
        convert_args += args
        self.convert(convert_args)
        return mnn_path
3. 转出的mnn配置文件修改cuda推理后端
{
    "llm_model": "llm.mnn",
    "llm_weight": "llm.mnn.weight",
    "backend_type": "cuda",
    "thread_num": 4,
    "precision": "low",
    "memory": "low",
    "mllm": {
        "backend_type": "cuda",
        "thread_num": 4,
        "precision": "low",
        "memory": "low"
    }
}

4. 修改部分源码，支持past_key_values分支输入推理，核心代码修改如下：
modules_[0].reset(Module::load(
      {"input_ids", "attention_mask", "position_ids", "past_key_values"},
      {"logits", "presents"}, model_path.c_str(), runtime_manager_,
      &module_config));

MNN::Express::VARP Llm::forwardRaw(MNN::Express::VARP hiddenState,
                                   MNN::Express::VARP mask,
                                   MNN::Express::VARP inputPos) {
  VARP logits;
  std::vector<MNN::Express::VARP> outputs;
  outputs = current_modules_.back()->onForward(
      {hiddenState, mask, inputPos, past_key_values_[0]});
  if (outputs.empty()) {
    return nullptr;
  }
  logits = outputs[0];
  past_key_values_[0] = outputs[1];
  //   mMeta->sync();
  return logits;
}

5. 将llm的推理改成cpu推理结果正确

![Image](https://github.com/user-attachments/assets/29631c71-c7de-464e-be7f-d64a8e94c88f)

6. 将llm的推理后端改为cuda后(详细配置见3)，推理结果异常

![Image](https://github.com/user-attachments/assets/f118c36d-54de-4b1f-a62b-c7db10cdacf4)
